composer.json

```diff
@@ -12,16 +12,16 @@
         }
     ],
     "require": {
-        "php": "^7.1.3",
-        "illuminate/config": ">=5.5 <6.0",
-        "illuminate/support": ">=5.5 <6.0",
-        "illuminate/database": ">=5.5 <6.0"
```

Comment:
Why did you dropped old versions of `illuminate` packages if there is breaking changes in the code? Your PR require to release major version of laravel-messenger.

Reply:
I thought it wouldn't work with PHP < 7.2. I've changed it back.